{{cookiecutter.project_name}}/.github/workflows/run-tests-fs.yml

```diff
@@ -0,0 +1,146 @@
+name: Feature and Training Integration Tests for {{cookiecutter.project_name}}
+on:
+  workflow_dispatch:
+  pull_request:
+    paths-ignore:
+      - 'databricks-config/**'
+
+env:
+  DATABRICKS_HOST: {{cookiecutter.databricks_staging_workspace_host}}
+  NODE_TYPE_ID: {{cookiecutter.cloud_specific_node_type_id}}
+  {% if cookiecutter.cloud == "aws" -%}
+  DATABRICKS_TOKEN: {% raw %}${{secrets.STAGING_WORKSPACE_TOKEN}}{% endraw %}
+  {%- endif %}
+
+concurrency: {{cookiecutter.project_name}}-feature-training-integration-test-staging
+
+jobs:
+  unit_tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install -r test-requirements.txt
+      - name: Run tests with pytest
+        run: pytest
+  
+  integration_test:
+    needs: unit_tests
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v3
+        {%- if cookiecutter.cloud == "azure" %}
+      - name: Generate AAD Token
+        {% raw %}run: ./.github/workflows/scripts/generate-aad-token.sh ${{ secrets.STAGING_AZURE_SP_TENANT_ID }} ${{ secrets.STAGING_AZURE_SP_APPLICATION_ID }} ${{ secrets.STAGING_AZURE_SP_CLIENT_SECRET }}{% endraw %}
+        {%- endif %}
+      - name: Fetch integration test
+        uses: actions/github-script@v6
+        id: integration-test-content
+        with:
+          script: |
+            const output = `
+                    {
+            "run_name": "features-training-integration-test",
+            "tasks": [
+              {
+                "task_key": "pickup-features",
+                "notebook_task": {
+                  "notebook_path": "notebooks/GenerateAndWriteFeatures",
+                  "base_parameters": {
+                    "env": "staging",
+                    "test_mode": "True",
+                    "input_table_path": "/databricks-datasets/nyctaxi-with-zipcodes/subsampled",
+                    "timestamp_column": "tpep_pickup_datetime",
+                    "output_table_name": "feature_store_taxi_example.trip_pickup_features-test",
+                    "features_transform_module": "pickup_features",
+                    "primary_keys": "zip"
+                  }
+                },
+                "existing_cluster_id": "0729-223229-zfjw22fr"
```

Comment:
Which cluster is this? We can't assume a cluster with this id exists everywhere the mlops stack is instantiated